### PR TITLE
feat(select-all): #CO-1756 select all ticket checkbox

### PR DESCRIPTION
**Pull request by Ozen40**
## Describe your changes
Checkbox that select all tickets
## Issue ticket number and link
[CO-1756](https://jira.support-ent.fr/browse/CO-1756)
## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [ ] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

File: src/main/resources/public/template/list-tickets.html [hunk 1]
```diff
@@ -13,6 +13,7 @@
 			<thead>
 				<tr>
                     <th>
+						<input type="checkbox" ng-change="selectAll()" ng-model="tickets.selected">
                     </th>
```
Comment (simonchaval) [inline src/main/resources/public/template/list-tickets.html:17]: Il faudrait revoir l'indentation de la balise <th> si possible

**Review by simonchaval — APPROVED** [review]

File: src/main/resources/public/ts/controllers/controller.ts [hunk 2]
```diff
@@ -918,5 +918,9 @@ export const SupportController: Controller = ng.controller('SupportController',
 			ticketService.exportSelectionCSV(model.getItemsIds($scope.tickets.selection()));
 		}
 
+		$scope.selectAll = () : void => {
+			$scope.tickets.all.forEach((ticket) => ticket.selected = !ticket.selected)
```
Comment (JinDouang) [inline src/main/resources/public/ts/controllers/controller.ts:922]: ;

Comment (JinDouang) [reply src/main/resources/public/ts/controllers/controller.ts:922]: A renommer la fonction car tu ne fais pas que selectAll, tu toggle plutôt

**Review by JinDouang — CHANGES_REQUESTED** [review]

**Review by BastienBenard — CHANGES_REQUESTED** [review]

File: src/main/resources/public/template/list-tickets.html [hunk 3]
```diff
@@ -1,108 +1,115 @@
...
-				<tr>
-                    <th>
-                    </th>
-					<th class="one centered" ng-click="switchSortBy('id')"
-					ng-class="{'order-by-reversed': (sort.expression === 'id' && sort.reverse === false),
+        <table class="list-view twelve" id="listTickets">
+            <thead>
+            <tr>
+                <th>
+                    <input type="checkbox" ng-change="toggleAll()" ng-model="tickets.selected">
```
Comment (Ozen40) [inline src/main/resources/public/template/list-tickets.html:16]: Modification ligne 16

**Pull request merged**